packages/frontend/src/utils/helper-api.js

```diff
@@ -2,13 +2,18 @@
 import * as nearApiJs from 'near-api-js';
 import { parseSeedPhrase } from 'near-seed-phrase';
 
+import { USE_INDEXER_SERVICE } from '../../../../features';
 import { ACCOUNT_HELPER_URL } from '../config';
+import { listAccountsByPublicKey } from '../services/indexer';
 
 export let controller;
```

Comment:
As far as I can tell this doesn't get used, so I've omitted it from the new call.